dds2val/README.md

```diff
@@ -0,0 +1,74 @@
+# DDS Provider
+
+The DDS provider provides data from an DDS middleware/API. For further understanding of the DDS middleware/API see [this](https://www.dds-foundation.org/what-is-dds-3/). The DDS provider only works with the KUKSA databroker. The KUKSA C++ server is not supported.
+
+## How to build
+
+### local build
+
+1. `python3 -m venv env && source env/bin/activate`
+2. `pip install -r requirements.txt`
+3. `chmod u+x ddsproviderlib/idls/generate_py_dataclass.sh`
+4. `./ddsproviderlib/idls/generate_py_dataclass.sh`
+
+### build image (suggested)
+
+1. `docker build -f Dockerfile --progress=plain --build-arg TARGETPLATFORM=linux/amd64 -t ddsprovider:latest .`
+
+### KML replay
+
+1. `python3 -m venv env && source env/bin/activate`
+2. `pip install -r requirements-kml.txt`
+3. `cd kml && python3 dds_kmlreplay.py directions.kml`
+
+## How to run
+
+Choose from local build or contanerization via docker.
+These steps are necessary:
+
+1. Run an instance of databroker aka: `docker run -it --rm --net=host ghcr.io/eclipse/kuksa.val/databroker:master`
+2. Start the KML replay with an active local python virtual environment: `pip install requirements/requirements-kml.txt && cd kml && python3 dds_kmlreplay.py directions.kml`
```

Comment:
This is almost the same as on line 20-22. If we are to keep it here we need to add ` -r` for `pip install`